presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -283,6 +284,11 @@ public SystemSessionProperties(
                         featuresConfig.isLegacyOrderBy(),
                         false),
                 booleanSessionProperty(
+                        SUPPORT_NULL_ON_LHS_OF_SEMI_JOIN,
```

Comment:
> It's not obvious for me that it is almost useless in practice. You can ...

Fair enough.

> I think for example I provided it is pushed correctly,

But, it it's correct, then what's the issue? It's been a while since I looked at this code and had a conversation about it, so I'm forgetting the details.